retriever/__main__.py

```diff
@@ -131,12 +131,33 @@ def main():
                 print("No scripts are currently available. Updating scripts now...")
                 check_for_updates(False)
                 print("\n\nScripts downloaded.\n")
-
-            if args.l is None and args.k is None:
-                all_scripts = dataset_names()
+            if not (args.l or args.k or (type(args.v) is list)):
+                all_scripts = datasetget_script_names()
```

Comment:
I think you might have accidentally inserted "get_script" into this. I tried it with `all_scripts = dataset_names()` and it seems to work fine. 

Otherwise, this looks real good!